lib/logstash/outputs/datadog.rb

```diff
@@ -82,7 +82,7 @@ def receive(event)
       request.add_field("Content-Type", 'application/json')
       response = @client.request(request)
       @logger.info("DD convo", :request => request.inspect, :response => response.inspect)
-      raise unless response.code == '200'
+      raise unless response.code == '202'
```

Comment:
maybe we should support any 2xx response here with:

```ruby
raise unless response.code ~= /^2\d\d$/
```

Reply:
That sounds like a more future-proof approach